Adding mapper configuration information (#3312)
kiegroup/kie-docs: doc-content/kogito-docs/src/main/asciidoc/dmn/chap-kogito-using-dmn-models.adoc

}
----

> Please note that for Json marshalling, Kogito is relying on https://quarkus.io/guides/rest-json#jackson[Quakus] and https://docs.spring.io/spring-boot/docs/1.3.0.M3/reference/html/howto-spring-mvc.html#howto-customize-the-jackson-objectmapper[SpringBoot] default mappers, which can be configured according to specific needs. 

Reviewer: ```suggestion
Note that for JSON marshaling, {PRODUCT} is relying on the default mappers of https://quarkus.io/guides/rest-json#jackson[Quarkus] and https://docs.spring.io/spring-boot/docs/1.3.0.M3/reference/html/howto-spring-mvc.html#howto-customize-the-jackson-objectmapper[SpringBoot], which can be configured according to the specific requirements. 
```
